Src/Plugins/Security/SEC1003.ReviewPotentiallySensitiveFiles.json

```diff
@@ -4,16 +4,111 @@
       "Name": "ReviewPotentiallySensitiveFiles",
       "Id": "SEC1003",
       "Level": "Error",
-      "FileNameAllowRegex": "(?i)\\.(pem|pkcs12|p12|pfx|asc|ovpn|rdp|mdf|sdf|sqlite|sqlite3|bek|tpm|fve|jks|psafe3|agilekeychain|kwallet|tblk|ppk)$",
+      "FileNameAllowRegex": "(?i)\\.(agilekeychain|asc|bek|fve|jks|kwallet|mdf|ovpn|p12|pem|pfx|pkcs12|ppk|psafe3|rdp|sdf|sqlite|sqlite3|tblk|tpm)$",
       "Description": "Review potentially sensitive files in versioned engineering content and other locations for security risk.",
       "Message": "'{0:scanTarget}' is {1:fileKind} that may compromise security if visible to attackers.",
-
       "MatchExpressions": [
+        {
+          "SubId": "1PasswordFile",
+          "FileNameAllowRegex": "(?i)\\.agilekeychain$",
```

Comment:
>agilekeychain$ [](start = 40, length = 14)

we should also flag .agilekeychain_zip files, apparently 1password supports compressing these and this is the naming convention